setup.cfg

```diff
@@ -53,7 +53,7 @@ install_requires =
     psutil>=5.0
     PyOpenGL>=3.1.0
     PyYAML>=5.1
-    pydantic>=1.7.3, !=1.8.0  # 1.8.0 broke something
+    pydantic>=1.7.3, !=1.8.0, !=1.8.1  # 1.8.0 broke something
```

Comment:
This change can now be removed. Master currently has `>=1.8.1` which is what we want it to be